postcss-selector-parser.d.ts

```diff
@@ -235,8 +235,8 @@ declare namespace parser {
         removeChild(child: Child): this;
         removeAll(): this;
         empty(): this;
-        insertAfter(oldNode: Child, newNode: Child): this;
-        insertBefore(oldNode: Child, newNode: Child): this;
+        insertAfter(oldNode: Child, ...newNode: Child[]): this;
+        insertBefore(oldNode: Child, ...newNode: Child[]): this;
```

Comment:
Type is wrong, it should be `insertAfter(oldNode: Child, newNode: Child, ...restNode[]): this;`

The first argument is required

Reply:
Done.